File: apzkr-pzpi-21-10-hotvianskyi-kyrylo/Task3-WebClient/src/interfaces/measurement.ts
export interface MeasurementDto {
    id: number,
    memberId: number,
    dateAndTime: string,
    height: number,
    weight: number,
    fatPercentage: number,
    musclePercentage: number,
    upperPressure: number,
    lowerPressure: number,
    bodyMassIndex: number,
    levelOfStress: number,
}

File: apzkr-pzpi-21-10-hotvianskyi-kyrylo/Task2-IoT/Domain/Measurement.cs
namespace Domain;

public class Measurement
{
    public double Height { get; set; }
    public double Weight { get; set; }
    public double FatPercentage { get; set; }
    public double MusclePercentage { get; set; }
    public int UpperPressure { get; set; }
    public int LowerPressure { get; set; }
    
    public double BodyMassIndex { get; set; }
    public double LevelOfStress { get; set; }
}